test.lua

```diff
@@ -280,143 +280,13 @@ init_box()
-
-test:test('simple expires test',  function(test)
-    test:plan(4)
-    -- put test tuples
-    log.info("-------------- put ------------")
-    log.info("put to space " .. prefix_space_id(space_id))
-    put_test_tuples(space_id, 10)
-
-    -- print before
-    log.info("------------- print -----------")
-    log.info("before print space " .. prefix_space_id(space_id), "\n")
-    print_test_tuples(space_id)
-    log.info("before print archive space " .. prefix_space_id(archive_space_id), "\n")
-    print_test_tuples(archive_space_id)
-
-    log.info("-------------- run ------------")
-    expirationd.start(
-        "test",
-        space_id,
-        check_tuple_expire_by_timestamp,
-        {
-            process_expired_tuple = put_tuple_to_archive,
-            args = {
-                field_no = 3,
-                archive_space_id = archive_space_id
-            },
-        }
-    )
-
-    -- wait expiration
-    local start_time = fiber.time()
-    log.info("------------- wait ------------")
-    log.info("before time = " .. os.date('%X', start_time))
-    fiber.sleep(5)
-    local end_time = fiber.time()
-    log.info("after time = " .. os.date('%X', end_time))
-
-    -- print after
-    log.info("------------- print -----------")
-    log.info("After print space " .. prefix_space_id(space_id), "\n")
-    print_test_tuples(space_id)
-    log.info("after print archive space " .. prefix_space_id(archive_space_id), "\n")
-    print_test_tuples(archive_space_id)
-
-    expirationd.tasks()
-
-    local task = expirationd.task("test")
-    test:is(task.start_time, start_time, 'checking start time')
-    test:is(task.name, "test", 'checking task name')
-    local restarts = 1
-    test:is(task.restarts, restarts, 'checking restart count')
-    local res = wait_cond(
-        function()
-            local task = expirationd.task("test")
-            local cnt = task.expired_tuples_count
-            return cnt == 7
-        end
-    )
-    test:is(res, true, 'Test task executed and moved to archive')
-    expirationd.kill("test")
-end)
-
-test:test("execution error test", function (test)
-    test:plan(2)
-    expirationd.start(
-        "test",
-        space_id,
-        check_tuple_expire_by_timestamp_error,
-        {
-            process_expired_tuple = put_tuple_to_archive,
-            args = {
-                field_no = 3,
-                archive_space_id = archive_space_id,
-            },
-        }
-    )
-    test:is(expirationd.task("test").restarts, 1, 'checking restart count')
-
-    expirationd.start("test",
-        space_id,
-        check_tuple_expire_by_timestamp,
-        {
-            process_expired_tuple = put_tuple_to_archive_error,
-            args = {
-                field_no = 3,
-                archive_space_id = archive_space_id,
-            },
-        }
-    )
-    local task = expirationd.task("test")
-    test:is(task.restarts, 1, 'Error task executed')
-    expirationd.kill("test")
-end)
```

Comment:
I do not understand why this test is needed, because there is no data and, accordingly, invalid validation functions are not called

Reply:
is_tuple_expired is not called in this test because no data available in space

Reply:
Added this
```lua
-- Check that task is alive and running.
helpers.retrying({}, function()
    t.assert_ge(full_scan_counter, 3)
end)
```